docs/dataservice-read-catalog-example.md

```diff
@@ -1,40 +1,47 @@
-# Read example
+# Read Example
 
-This example shows how to retrieve catalog configuration, partition metadata, and partition data using the HERE OLP SDK for C++.
+On this page, find instructions on how to build and run the example project on different platforms and get catalog and partition metadata, as well as partition data using the HERE Open Location Platform (OLP) SDK for C++.
 
-Before you run the example, authorize by replacing the placeholders in `examples/dataservice-read/example.cpp` with your app access key id and access key secret that you can find on the platform `Apps & keys` page:
+Before you run the example, authorize to HERE OLP:
 
-```cpp
-const std::string kKeyId(""); // your here.access.key.id
-const std::string kKeySecret(""); // your here.access.key.secret
-```
+1. On the [Apps & keys](https://platform.here.com/admin/apps) page, copy your application access key ID and access key secret.
+   For instructions on how to get the access key ID and access key secret, see the [Get Credentials](https://developer.here.com/olp/documentation/access-control/user-guide/topics/get-credentials.html) section in the Terms and Permissions User Guide.
 
-## Building and running on Linux
+2. In `examples/dataservice-read/example.cpp`, replace the placeholders with your access key ID and access key secret.
 
-Configure the project with `OLP_SDK_BUILD_EXAMPLES` set to `ON` to enable examples `CMake` targets:
+   ```cpp
+   const std::string kKeyId(""); // your here.access.key.id
+   const std::string kKeySecret(""); // your here.access.key.secret
+   ```
 
-```bash
-mkdir build && cd build
-cmake -DOLP_SDK_BUILD_EXAMPLES=ON ..
-```
+## Build and Run on Linux
 
-To build the example, run the following command in the `build` folder:
+To build and run the example on Linux:
 
-```bash
-cmake --build . --target dataservice-read-example
-```
+1. Enable examples of the `CMake` targets.
 
-To execute the example, run:
+   ```bash
+   mkdir build && cd build
+   cmake -DOLP_SDK_BUILD_EXAMPLES=ON ..
+   ```
 
-```bash
-./examples/dataservice-read/dataservice-read-example
-```
+2. In the **build** folder, build the example.
+
+   ```bash
+   cmake --build . --target dataservice-read-example
+   ```
+
+3. Execute the example.
+
+   ```bash
+   ./examples/dataservice-read/dataservice-read-example
+   ```
 
-After building and running the example, you see the following:
+After building and running the example, you see the following information:
 
-* `edge-example-catalog` as a catalog description
-* `versioned-world-layer` as a layer description
-* `Request partition data - Success, data size - 3375`
+- `edge-example-catalog` &ndash; a catalog description
+- `versioned-world-layer` &ndash; a layer description
+- `Request partition data - Success, data size - 3375` - a success message with an amount of data size
```

Comment:
```suggestion
- `Request partition data - Success, data size - 3375` - a success message which displays the size of retrieved data from partition
```
I think better be more explicit about the data size message

Reply:
Fixed